docs/eog/case-studies-cw20-infrastructure.rst

```diff
@@ -0,0 +1,67 @@
+.. _CW20-Infrastructure: 
+
+CW20 Infrastructure
+====================
+
+We had already organised registration for the in-person event through `Eventbrite <https://www.eventbrite.com/>`_. 
+We sent emails to registrants through Eventbrite to notify them of our decision making every step of the way. 
+Because of the unconference nature of the event, we even asked for participant input to our contingency investigation. 
+As we worked to reformat the event and move it online, we gave registrants time to cancel and collect full refunds. 
+We also reduced the cost of the event (from £250 to £50) and issued partial refunds through Eventbrite to participants who wanted to stay and participate in the virtual event. 
+We took the partial refund route in order to maintain continuity with our registrants, but another option could have been to outright cancel the Eventbrite event, issue full refunds and start from scratch. 
+We also provided `guidance <https://software.ac.uk/news/collaborations-workshop-2020-cw20-move-online>`_ and documentation on reclaiming costs to help participants who had already booked their travel to Belfast.
+
+For the in-person event, we had planned to use additional tools such as `Slack <https://slack.com>`_, `Slido <https://www.sli.do/>`_ and `uCONFLY <http://uconfly.org/>`_, however for the online event we felt that it was important to minimise the number of platforms that participants had to register with and keep track of. 
+We narrowed our needs down to platforms for video conferencing, a chat system for sharing information and community engagement, and a tool for collaborative note-taking to keep everyone synchronised. 
+
+Platforms
+----------
+
+Video conferencing
+^^^^^^^^^^^^^^^^^^^
+
+`Zoom <https://zoom.us/>`_ was very quickly identified as the video conferencing platform that we would use because we had the most experience with it and it had the functionality we needed (namely breakout rooms and easy recording of the event). 
```

Comment:
```suggestion
`Zoom <https://zoom.us/>`_ was very quickly identified as the video conferencing platform that we would use because we had the most experience with it and it had the functionality we needed (scaling and stability with many video based participants, breakout rooms and easy recording of the event). 
```